gltf-json/src/scene.rs

```diff
@@ -66,16 +75,26 @@ pub struct Node {
     pub weights: Option<Vec<f32>>,
 }
 
-fn node_scale_default() -> [f32; 3] {
-    [1.0, 1.0, 1.0]
+impl Node {
```

Comment:
I'd rather these default values be written inline for the same reasons as `byte_offset`.